website/util/sanitize.py

```diff
@@ -94,3 +97,16 @@ def safe_unescape_html(value):
             value = value.replace(escape_sequence, character)
         return value
     return value
+
+
+def safe_json(value):
```

Comment:
I'd like to change this to `to_json`, `as_json`, or something similar - both here and in `get_globals` in `routes.py`.

Of the two changes, I feel most strongly about the naming inside of a Mako template context. The function itself can have a more explicit name if that makes more sense.